config/locales/eng/action.yml

```diff
@@ -2260,6 +2260,7 @@ eng:
         other: "There are %{count} entries to point"
       you_must_have_an_email_column: "You must have an email address in your extraction"
       you_need_to_review_your_application_configuration: "You need to review your application configuration"
+      locking_vesrsion_error: "Sorry, please try after someone time, someone else editing this working version elemen"
```

Comment:
Translation key must be fixed too and text should be: "Sorry, please retry in a moment, someone else is updating this record"